k9mail/src/main/java/com/fsck/k9/widget/list/MessageListWidgetProvider.java

```diff
@@ -0,0 +1,85 @@
+package com.fsck.k9.widget.list;
+
+
+import android.app.PendingIntent;
+import android.appwidget.AppWidgetManager;
+import android.appwidget.AppWidgetProvider;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.widget.RemoteViews;
+
+import com.fsck.k9.R;
+import com.fsck.k9.activity.MessageCompose;
+import com.fsck.k9.activity.MessageList;
+
+
+public class MessageListWidgetProvider extends AppWidgetProvider {
+    private static String ACTION_UPDATE_MESSAGE_LIST = "UPDATE_MESSAGE_LIST";
+
+
+    public static void updateMailViewList(Context context) {
```

Comment:
This name doesn't indicate that the method doesn't actually update anything itself, but just sends a broadcast intent